### PR TITLE
Cell library and family info script updates

### PR DESCRIPTION
**Pull request by ttown523**
In this commit:
----------------

- Resolves issue #44

### CODE REVIEW

File: tincr/io/library/genCellLibrary.tcl [hunk 1]
```diff
@@ -853,6 +853,10 @@ proc ::tincr::create_xml_cell_library { {part xc7a100t-csg324-3} {filename ""} {
     # Write the cell library xml file header
     puts $xml_out {<?xml version="1.0" encoding="UTF-8"?>}
     puts $xml_out "<root>"
+    
+    # Add the family tag to the xml file  
+    puts $xml_out "  <family>[string toupper [get_property FAMILY [get_parts -of [get_design]]]]</family>"
```
Comment (bradselw) [inline tincr/io/library/genCellLibrary.tcl:858]: Prefer the explicit `-of_objects` to the potentially ambiguous `-of`

File: tincr/io/library/genFamilyInfo.tcl [hunk 2]
```diff
@@ -91,7 +91,8 @@ proc uniqueSiteLists {primary_name alternate_name compatible_name} {
 	# Create a dictionary of sites that are only alternates
 	set alternates [dict create]
 	dict for {type site} $alternate_tmp {
-		if {![dict exists $primary $type] &&  ![regexp {.*IOB*} $type]} {
+		
+        if {![dict exists $primary $type] &&  ![regexp {.*IOB*} $type]} {
```
Comment (bradselw) [inline tincr/io/library/genFamilyInfo.tcl:94]: Indentation is off

**Review by bradselw — APPROVED** [review]
Couple nits, other than that looks good

File: tincr/io/library/genFamilyInfo.tcl (hunk 2 above)
Comment (ttown523) [reply tincr/io/library/genFamilyInfo.tcl:94]: hmm, that is weird. the indentation is right in my IDE and notpad++.

**Pull request merged**